test/e2e/retistry_provisioner_test.go

```diff
@@ -0,0 +1,78 @@
+package e2e
```

Comment:
nit: there is a typo in the name of the test file. It should be `registry_provisioner_test`

Reply:
Good catch!  Thanks!